### PR TITLE
Promo component

### PR DESCRIPTION
**Pull request by sabi98**

### CODE REVIEW

**Comment by AdamChrimes** [comment]
I'm getting a error when running the migrations locally

```
(wagtail-nhsuk-frontend) bash-3.2$ python3 manage.py migrate
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, home, sessions, taggit, wagtailadmin, wagtailcore, wagtaildocs, wagtailembeds, wagtailforms, wagtailimages, wagtailnhsukfrontendsettings, wagtailredirects, wagtailsearch, wagtailusers
Running migrations:
  Applying home.0008_hubspage...Traceback (most recent call last):
  File "/Users/adamchrimes/.local/share/virtualenvs/wagtail-nhsuk-frontend-Hsp7880J/lib/python3.7/site-packages/django/db/backends/utils.py", line 83, in _execute
    return self.cursor.execute(sql)
  File "/Users/adamchrimes/.local/share/virtualenvs/wagtail-nhsuk-frontend-Hsp7880J/lib/python3.7/site-packages/django/db/backends/sqlite3/base.py", line 296, in execute
    return Database.Cursor.execute(self, query)
sqlite3.OperationalError: table "home_hubspage" already exists
``

**Comment by sabi98** [comment]
> The promo was imported in models.py but it wasnt put in the streamfield

It gets added to the hubs page at the bottom.

**Comment by Ross-Clark** [comment]
do the heading levels need to be variable @AdamChrimes

**Comment by AdamChrimes** [comment]
Yeah @Ross-Clark they need to be customizable levels, similar to the other ones.

**Comment by AdamChrimes** [comment]
The component looks good to me now the template is working. Just needs the header level customisation and all good.

File: testapp/home/templates/home/hubs_page.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+  {% block main %}
```
Comment (AdamChrimes) [inline testapp/home/templates/home/hubs_page.html]: @sabi98 Mike is going to get you for this indenting

Comment (mikemonteith) [reply testapp/home/templates/home/hubs_page.html]: 👀 👿

**Comment by mikemonteith** [comment]
I've added some test data and refactored a few things.
Probably need a review from someone else now 😄

**Pull request merged**